index.html

```diff
@@ -161,10 +161,10 @@ <h2 class="cards-title">Main Program</h2>
               </p>
             </div>
           </div>
-        </div>
-        <div class="cards-bottom-line">
-          <h3 class="card-see-more">SEE THE WHOLE PROGRAM</h3>
-          <hr class="line-breaker-white" />
+          <div class="cards-bottom-line">
+            <h3 class="card-see-more">SEE THE WHOLE PROGRAM</h3>
+            <hr class="line-breaker-white" />
+          </div>
         </div>
       </section>
```

Comment:
- [ ] Please as I said in the previous comment, you must stick to the original design as much as you can. Kindly add the `more` button below the `speakers`.
- Your design:
<img width="206" alt="image" src="https://user-images.githubusercontent.com/77190203/159141753-36a5b0a0-7f8f-40b4-a471-8f041f163b7d.png">

- Original design:
<img width="208" alt="image" src="https://user-images.githubusercontent.com/77190203/159141770-b399a72b-b81f-4f26-93a7-8c744f57ce0d.png">